package.json

```diff
@@ -1,13 +1,19 @@
 {
   "name": "@hawk.so/nodejs",
-  "version": "3.1.2",
+  "version": "3.1.3",
   "description": "Node.js catcher for Hawk",
   "license": "MIT",
   "main": "./dist/src/index.js",
-  "types": "./dist/src/index.d.ts",
+  "module": "dist/mjs/index.js",
+  "exports": {
+    ".": {
+        "import": "./dist/mjs/index.js",
+        "require": "./dist/cjs/index.js"
```

Comment:
I tried to import this but wont work.
But change above as follow and worked.
```
"import": "./dist/mjs/src/index.js",
 "require": "./dist/cjs/src/index.js"
```

Reply:
Thanks, fixed